bundle/manifests/splunk-operator.clusterserviceversion.yaml

```diff
@@ -911,7 +914,7 @@ spec:
     name: Splunk Inc.
     url: www.splunk.com
   relatedImages:
-  - image: splunk/splunk:9.0.1-a1
+  - image: docker.io/splunk/splunk:edge
```

Comment:
Same here.

Reply:
this is an autogenerated file. changes need to be set to 9.1 splunk release